thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionTaskRunner.java

```diff
@@ -91,6 +98,10 @@ public DetectionTaskRunner() {
     Map<DimensionKey, MetricTimeSeries> dimensionKeyMetricTimeSeriesMap =
         TimeSeriesUtil.getTimeSeriesForAnomalyDetection(anomalyFunctionSpec, startEndTimeRanges);
 
+    overrideConfigs = OverrideConfigHelper.getTimeSeriesOverrideConfigs(
```

Comment:
I see, I dont see OverrideConfigManagerImpl class as part of this PR. Is that already checkedin?